### PR TITLE
adding readme for samples

### PR DESCRIPTION
**Pull request by hemanttanwar**
1. Identity README : Fix API changes in example
2. KV-secret : move samples section into sample/README.md

### CODE REVIEW

File: sdk/keyvault/azure-security-keyvault-secrets/src/samples/README.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+## Getting started
+
+Typically, you will not need to install or specifically depend on Azure Core, instead it will be transitively downloaded by your build tool when you depend on of the client libraries using it. In case you want to depend on it explicitly (to implement your own client library, for example), include the following Maven dependency:
+
+[//]: # ({x-version-update-start;com.azure:azure-core;current})
+```xml
+<dependency>
+  <groupId>com.azure</groupId>
+  <artifactId>azure-core</artifactId>
+  <version>1.0.0</version>
```
Comment (g2vinay) [inline sdk/keyvault/azure-security-keyvault-secrets/src/samples/README.md]: Why do we need this here ?

File: sdk/keyvault/azure-security-keyvault-secrets/src/samples/README.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
+
+# Samples Azure Core APIs
```
Comment (g2vinay) [inline sdk/keyvault/azure-security-keyvault-secrets/src/samples/README.md]: Why Azure Core APIs ?

File: sdk/keyvault/azure-security-keyvault-secrets/src/samples/README.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+    * Recover a deleted secret
+    * Purge Deleted secret
+
+<!-- LINKS -->
+[source_code]:  src
+[sample_helloWorld]: src/samples/java/com/azure/security/keyvault/secrets/HelloWorld.java
+[sample_helloWorldAsync]: src/samples/java/com/azure/security/keyvault/secrets/HelloWorldAsync.java
+[sample_list]: src/samples/java/com/azure/security/keyvault/secrets/ListOperations.java
+[sample_listAsync]: src/samples/java/com/azure/security/keyvault/secrets/ListOperationsAsync.java
+[sample_BackupRestore]: src/samples/java/com/azure/security/keyvault/secrets/BackupAndRestoreOperations.java
```
Comment (g2vinay) [inline sdk/keyvault/azure-security-keyvault-secrets/src/samples/README.md]: These relative links will not work as it is.

Comment (g2vinay) [reply sdk/keyvault/azure-security-keyvault-secrets/src/samples/README.md]: Need to be updated, since the file location has moved,

**Comment by hemanttanwar** [comment]
Fixes #5935

**Comment by joshfree** [comment]
The repo's root README.md also needs a s/October/November/g

File: sdk/keyvault/azure-security-keyvault-secrets/README.md [hunk 4]
```diff
@@ -381,5 +355,6 @@ This project has adopted the [Microsoft Open Source Code of Conduct](https://ope
 [sample_BackupRestoreAsync]: src/samples/java/com/azure/security/keyvault/secrets/BackupAndRestoreOperationsAsync.java
 [sample_ManageDeleted]: src/samples/java/com/azure/security/keyvault/secrets/ManagingDeletedSecrets.java
```
Comment (srnagar) [inline sdk/keyvault/azure-security-keyvault-secrets/README.md]: These links should now be deleted as they are moved to samples/README.md

File: sdk/keyvault/azure-security-keyvault-secrets/src/samples/README.md [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+### Cryptography Client:
+The Cryptography client performs the cryptographic operations locally or calls the Azure Key Vault service depending on how much key information is available locally. It supports encrypting, decrypting, signing, verifying, key wrapping, key unwrapping and retrieving the configured key. An asynchronous and synchronous, CryptographyClient, client exists in the SDK allowing for selection of a client based on an application's use case.
+
+
+# Samples Azure Key Vault secrets APIs
+This document describes how to use samples and what is done in each sample.
+
+## Getting started
+
+Typically, you will not need to install or specifically depend on Azure Core, instead it will be transitively downloaded by your build tool when you depend on of the client libraries using it. In case you want to depend on it explicitly (to implement your own client library, for example), include the following Maven dependency:
```
Comment (srnagar) [inline sdk/keyvault/azure-security-keyvault-secrets/src/samples/README.md]: ".. depend on one of the client libraries using it."

File: sdk/keyvault/azure-security-keyvault-secrets/src/samples/README.md [hunk 6]
```diff
@@ -0,0 +1,112 @@
...
+[sample_helloWorld]: java/com/azure/security/keyvault/secrets/HelloWorld.java
+[sample_helloWorldAsync]: java/com/azure/security/keyvault/secrets/HelloWorldAsync.java
+[sample_list]: java/com/azure/security/keyvault/secrets/ListOperations.java
+[sample_listAsync]: java/com/azure/security/keyvault/secrets/ListOperationsAsync.java
+[sample_BackupRestore]: java/com/azure/security/keyvault/secrets/BackupAndRestoreOperations.java
+[sample_BackupRestoreAsync]: java/com/azure/security/keyvault/secrets/BackupAndRestoreOperationsAsync.java
+[sample_ManageDeleted]: java/com/azure/security/keyvault/secrets/ManagingDeletedSecrets.java
+[sample_ManageDeletedAsync]: java/com/azure/security/keyvault/secrets/ManagingDeletedSecretsAsync.java
+
+![Impressions](https://azure-sdk-impressions.azurewebsites.net/api/impressions/azure-sdk-for-java/sdk/core/azure-core/README.png)
```
Comment (g2vinay) [inline sdk/keyvault/azure-security-keyvault-secrets/src/samples/README.md]: This link reads azure-core ?
Needs to be fixed.

**Pull request merged**